crypto/src/main/java/org/springframework/security/crypto/key/SymmetricKey.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.crypto.key;
+
+import java.util.Map;
+import java.util.UUID;
+
+import javax.crypto.SecretKey;
+
+/**
+ * A {@link CryptoKey} that holds a {@code javax.crypto.SecretKey} used for symmetric
+ * algorithm's.
```

Comment:
Not to take away from the more important discussion about contracts, but it would be clearer to readers if the apostrophe were only used when talking about possession.

Here, for example, `used for symmetric algorithms`, is correct. There are a few other places in the PR where the possessive ' is being used to pluralize.